cmd/launcher/launcher.go

```diff
@@ -71,13 +71,14 @@ import (
 
 const (
 	// Subsystems that launcher listens for control server updates on
-	agentFlagsSubsystemName  = "agent_flags"
-	serverDataSubsystemName  = "kolide_server_data"
-	desktopMenuSubsystemName = "kolide_desktop_menu"
-	authTokensSubsystemName  = "auth_tokens"
-	katcSubsystemName        = "katc_config" // Kolide ATC
-	ztaInfoSubsystemName     = "zta_info"    // legacy name for dt4aInfo subsystem
-	dt4aInfoSubsystemName    = "dt4a_info"
+	agentFlagsSubsystemName       = "agent_flags"
+	serverDataSubsystemName       = "kolide_server_data"
+	desktopMenuSubsystemName      = "kolide_desktop_menu"
+	authTokensSubsystemName       = "auth_tokens"
+	katcSubsystemName             = "katc_config" // Kolide ATC
+	ztaInfoSubsystemName          = "zta_info"    // legacy name for dt4aInfo subsystem
+	dt4aInfoSubsystemName         = "dt4a_info"
+	osqueryVariablesSubsystemName = "osquery_variables"
```

Comment:
I don't know what we want to call these. My hunch is that it's really much closer to "Server Data" than the thing we've already named that. (That other thing is really closer to "Device Data"

"Osquery Variables" might be okay, it's a little jargon." Shared Server Data" might be okay (this is shared across pretty much everything). dunno really. What would be clear to someone in a year?